docs/plugins/google-tagmanager/index.js

```diff
@@ -20,19 +20,14 @@ module.exports = function (context, options) {
     );
   }
 
-  const isProd = process.env.NODE_ENV === 'production';
-
   return {
     name: 'google-tagmanager',
 
     getClientModules() {
-      return isProd ? [path.resolve(__dirname, './client')] : [];
+      return [path.resolve(__dirname, './client')];
     },
 
     injectHtmlTags() {
-      if (!isProd) {
```

Comment:
why do we want to include Klaro in dev/demo versions?

Reply:
This is just because I need a way to test it locally. Results from non-production domains are still filtered out in GA though

Reply:
ok 👍